docs/en/writing-running-appium/android/activity-startup.md

```diff
@@ -0,0 +1,34 @@
+## How To Troubleshoot Activities Startup
+
+
+### Capabilities
+
+`The Activity class is a crucial component of an Android app, and the way activities are launched and put together is a fundamental part of the platform's application model. Unlike programming paradigms in which apps are launched with a main() method, the Android system initiates code in an Activity instance by invoking specific callback methods that correspond to specific stages of its lifecycle.` (c) [Android Developer Documentation](https://developer.android.com/guide/components/activities/intro-activities)
```

Comment:
I'd rather this be blockquote.

### Capabilities
> The Activity class is a crucial component of an Android app, and the way activities are launched and put together is a fundamental part of the platform's application model. Unlike programming paradigms in which apps are launched with a main() method, the Android system initiates code in an Activity instance by invoking specific callback methods that correspond to specific stages of its lifecycle.
> &copy; [Android Developer Documentation](https://developer.android.com/guide/components/activities/intro-activities)

Appium needs to know package and activity names in order to properly initialize the application under test. This information is expected to be provided in driver capabilities and consists of the following keys:
...


This would be achieved by:
```markdown
### Capabilities
> The Activity class is a crucial component of an Android app, and the way activities are launched and put together is a fundamental part of the platform's application model. Unlike programming paradigms in which apps are launched with a main() method, the Android system initiates code in an Activity instance by invoking specific callback methods that correspond to specific stages of its lifecycle.
> &copy; [Android Developer Documentation](https://developer.android.com/guide/components/activities/intro-activities)

Appium needs to know package and activity names in order to properly initialize the application under test. This information is expected to be provided in driver capabilities and consists of the following keys:
```

Reply:
fixed